content/includes/1-topic-landing.html

```diff
@@ -1,23 +1,26 @@
 <div class="main" role="main">
 {% unless page.template == 'alt-1-topic-landing' %}
 
-  {% if collection == 'disability-benefits' %}
-    {% include action-bar.html action_text="Apply for Disability Benefits" action_url="/disability-benefits/apply-for-benefits/" action_class="va-action-bar--header" %}
+  {% if path contains 'disability-benefits' %}
```

Comment:
Is this going to cause a problem with substrings?

Like  `/disability-benefits-exceptions/` would match this as well...